CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - **CUMULUS-2312** - RDS Migration Epic Phase 3
+  - **CUMULUS-2810**
```

Comment:
Consider noting the translation bugfix as well?

Reply:
Yes.  Added to merge feature-rds3 into this branch a359e536d7